ui/jobs/components/ast.ts

```diff
@@ -90,7 +101,7 @@ export class ASTComponent extends BaseComponent {
         this.combustBox.duration = 18;
         break;
       case kAbility.Draw:
-        this.drawBox.duration = 30;
+        this.drawBox.duration = 30 + this.drawBox.value;
```

Comment:
Does this need to be behind an `if (this.is5x) {`?

Reply:
You are right. I think this will not make a difference at first, but then I come to "Lost Font of Skill"...